src/ralph/lib/transitions/conf.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
 
-
+DEFAULT_ASYNC_TRANSITION_SERVICE_NAME = 'ASYNC_TRANSITIONS'
```

Comment:
https://github.com/allegro/ralph/blob/feature/deployment/src/ralph/settings/base.py#L289 - it is stored in ralph settings as well